hana/lib/HANAService.js

```diff
@@ -6,6 +6,12 @@ const { SQLService } = require('@cap-js/db-service')
 const drivers = require('./drivers')
 const cds = require('@sap/cds')
 const collations = require('./collations.json')
+const keywords = require('@sap/cds-compiler').to.hdi.keywords
```

Comment:
with that we have introduced a dependency to compiler at runtime. So far we did not have that. We either need to declare it or keep the local list. If this is all we need, I'd vote for a local list.

Reply:
dont we have the dependency in any case due to the dependency to `sap/cds`?

(I think `cds` also exports the keywords, we could use the cds api instead)

Reply:
If cds exports it, that's fine.
I was just thinking that we introduce a dependency to the compiler only for keywords which seems like an overkill to me.

Reply:
→ #530